test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -336,13 +338,6 @@ public void PushCommand_PushToServerBasicAuth()
                     using (var server = new MockServer())
                     {
                         server.Listener.AuthenticationSchemes = AuthenticationSchemes.Basic;
-                        server.Get.Add("/nuget", r =>
```

Comment:
I think it is fine, lets file a tracking bug and immediately close it as won't fix with justification (that this was a hack for old fwlink nuget.org used to do).